.swiftlint.yml

```diff
@@ -1,12 +1,136 @@
+#
+# Automatically downloaded from https://raw.githubusercontent.com/raywenderlich/swift-style-guide/master/com.raywenderlich.swiftlint.yml by SwiftLint on 22/08/2022 at 4:40:02 pm.
```

Comment:
Just use parent_config as discussed.

Reply:
Ignore me you have updated below. 👍

Reply:
Personally I would actually favour not copying the remote style guide. Not sure how I feel about the chance of us getting styling warnings / errors just of the back of someone else's questionable decisions 🤔

Reply:
We did the same on Checkout iOS SDK to keep ourselves updated with new changes. Yes this could break things but I am sure a few people maintain it so some thought is put into when making new rules. Upside is we get latest updates. If we plan to change to this we need to change it on checkout iOS SDK as well. For now I am fine to have it referring to remote guide.

Reply:
> Upside is we get latest updates

I'd like to hope we'll grow the team to a level we can enable engineers to lead on "latest updates" rather than just following what some 3rd party with questionable preferences decides to be "latest updates".